spec/lex.dd

```diff
@@ -332,11 +358,33 @@ $(GNAME StringPostfix):
     $(B d)
 
 $(GNAME DelimitedString):
-    $(B q") $(I Delimiter) $(GLINK WysiwygCharacters)$(OPT) $(I MatchingDelimiter) $(B ")
+    $(B q") $(GLINK Delimiter) $(GLINK WysiwygCharacters)$(OPT) $(GLINK MatchingDelimiter) $(B ")
+
+$(GNAME Delimiter):
+    $(B $(LPAREN))
+    $(B {)
+    $(B [)
+    $(B <)
+    $(GLINK Identifier)
+
+$(GNAME MatchingDelimiter):
+    $(B $(RPAREN))
+    $(B })
+    $(B ])
+    $(B >)
+    $(GLINK Identifier))
```

Comment:
This rule could be modified to enforce the correct delimiters for almost all combinations except `Identifier`.